lang/de.yml

```
@@ -108,14 +108,14 @@ LOOKUP_BLOCK: "{0} {platziert|entfernt} {1}."
 LOOKUP_CONTAINER: "{0} {hinzugefügt|entfernt} {1} {2}."
 LOOKUP_HEADER: "{0} Ergebnisse"
 LOOKUP_INTERACTION: "{0} {angeklickt|getötet} {1}."
-LOOKUP_ITEM: "{0} {abgeholt|abgeworfen} {1} {2}."
-LOOKUP_LOGIN: "{0} eingeloggt {rein|raus}."
+LOOKUP_ITEM: "{0} {aufgehoben|abgeworfen} {1} {2}."
+LOOKUP_LOGIN: "{0} {ein|aus}geloggt."
 LOOKUP_PAGE: "Seite {0}"
 LOOKUP_PROJECTILE: "{0} {geworfen|geschossen} {1} {2}."
 LOOKUP_ROWS_FOUND: "{0} {Reihe|Reihen} gefunden."
 LOOKUP_SEARCHING: "Suche nachschlagen. Warten Sie mal..."
 LOOKUP_STORAGE: "{0} {hinterlegt|abgehoben} {1} {2}."
-LOOKUP_TIME: "{0} vor"
+LOOKUP_TIME: "Vor {0}"
```

Personally I think it looks better in-game if it's lowercase like in English, although it doesn't really matter

```suggestion
LOOKUP_TIME: "vor {0}"
```